soccer/planning/RRTPlanner.cpp

```diff
@@ -137,15 +139,24 @@ Planning::InterpolatedPath* RRTPlanner::run(
 	{
 		// FIXME: without these two lines, an empty path is returned which causes errors down the line.
 		path->points.push_back(start);
+		path->times.push_back(0);
+		path->vels.push_back(Geometry2d::Point(0,0));
 	}
 	return path;
 }
-
+Planning::InterpolatedPath update(
```

Comment:
Its a skeleton method for when we can use these classes to replan from a certain point in a path.